src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Windows/UnoQuickStart.Windows.csproj

```diff
@@ -20,32 +20,27 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.2.221109.1" />
```

Comment:
Why did you remove this one? It comes transitively, but it's generally best to have the explicit dependency for users to upgrade.